compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -129,6 +129,16 @@ unsafe fn configure_llvm(sess: &Session) {
 
     llvm::LLVMInitializePasses();
 
+    for plugin in &sess.opts.debugging_opts.llvm_plugins {
+        let path = CString::new(plugin.as_bytes()).unwrap();
+        let res = libc::dlopen(path.as_ptr(), libc::RTLD_LAZY | libc::RTLD_GLOBAL);
+        if res.is_null() {
+            println!("{}", CStr::from_ptr(libc::dlerror()).to_string_lossy().into_owned());
+        }
+        println!("{:p}", res);
```

Comment:
No debug `println`s in the compiler codebase, please. Consider `debug!`.

Reply:
Done